shortcodes/ucf-degree-list-shortcode.php

```diff
@@ -28,7 +29,21 @@ public static function shortcode( $atts ) {
 				'order'          => 'ASC'
 			);
 
-			if ( $atts['filter_by_tax'] && $atts['terms'] ) {
+			if ( $atts['filter_by_taxonomies'] && $atts['terms'] ) {
```

Comment:
I agree with what Jim's talking about with repurposing the `filter_by_tax` option instead of creating a separate one here that effectively achieves the same thing.